addons/hr_expense/models/hr_expense.py

```diff
@@ -644,9 +675,16 @@ def set_to_paid(self):
         self.write({'state': 'done'})
 
     @api.multi
-    def approve_expense_sheets(self):
+    def approve_sheet(self):
         if not self.user_has_groups('hr_expense.group_hr_expense_user'):
             raise UserError(_("Only HR Officers can approve expenses"))
+        elif self.employee_id.user_id.id == self.env.user.id and not self.user_has_groups('hr_expense.group_hr_expense_manager'):
```

Comment:
Small note: the 'id' suffix is not necessary as we can directly compare record sets.